packages/destination-actions/src/destinations/snap-conversions-api/snap-capi-properties.ts

```diff
@@ -0,0 +1,327 @@
+    { label: 'Mobile App', value: 'MOBILE_APP' }
+  ]
+}
+
+export const event_tag: InputField = {
+  label: 'Event Tag',
+  description: 'Custom event label.',
+  type: 'string'
+}
+
+export const timestamp: InputField = {
+  label: 'Event Timestamp',
+  description:
+    'The Epoch timestamp for when the conversion happened.  The timestamp cannot be more than 28 days in the past',
+  type: 'string',
+  default: {
+    '@path': '$.timestamp'
+  }
+}
+
+export const email: InputField = {
+  label: 'Email',
+  description:
+    'Email address of the user who triggered the conversion event. Segment will normalize and hash this value before sending to Snapchat.',
+  type: 'string',
+  default: {
+    '@if': {
+      exists: { '@path': '$.properties.email' },
+      then: { '@path': '$.properties.email' },
+      else: { '@path': '$.traits.email' }
+    }
+  }
+}
+
+export const mobile_ad_id: InputField = {
+  label: 'Mobile Ad Identifier',
+  description:
+    'Mobile ad identifier (IDFA or AAID) of the user who triggered the conversion event. Segment will normalize and hash this value before sending to Snapchat.',
+  type: 'string',
+  default: {
+    '@path': '$.context.device.advertisingId'
+  }
+}
+
+export const uuid_c1: InputField = {
+  label: 'uuid_c1 Cookie',
+  description:
+    'Unique user ID cookie. If you are using the Pixel SDK, you can access a cookie1 by looking at the _scid value.',
+  type: 'string'
+}
+
+export const idfv: InputField = {
+  label: 'Identifier for Vendor',
+  description: 'IDFV of the user’s device. Segment will normalize and hash this value before sending to Snapchat.',
+  type: 'string',
+  default: {
+    '@path': '$.context.device.id'
+  }
+}
+
+export const phone_number: InputField = {
+  label: 'Phone Number',
+  description:
+    'Phone number of the user who triggered the conversion event. Segment will normalize and hash this value before sending to Snapchat.',
+  type: 'string',
+  default: {
+    '@if': {
+      exists: { '@path': '$.properties.phone' },
+      then: { '@path': '$.properties.phone' },
+      else: { '@path': '$.traits.phone' }
+    }
+  }
+}
+
+export const user_agent: InputField = {
+  label: 'User Agent',
+  description: 'User agent from the user’s device.',
+  type: 'string',
+  default: {
+    '@path': '$.context.userAgent'
+  }
+}
+
+export const ip_address: InputField = {
+  label: 'IP Address',
+  description:
+    'IP address of the device or browser. Segment will normalize and hash this value before sending to Snapchat.',
+  type: 'string',
+  default: {
+    '@path': '$.context.ip'
+  }
+}
+
+export const item_category: InputField = {
+  label: 'Item Category',
+  description: 'Category of the item.',
+  type: 'string',
+  default: {
+    '@path': '$.properties.category'
+  }
+}
+
+export const item_ids: InputField = {
+  label: 'Item IDs',
+  description: 'International Article Number (EAN) when applicable, or other product or category identifier. ',
```

Comment:
Remove the trailing whitespace.
```suggestion
  description: 'International Article Number (EAN) when applicable, or other product or category identifier.',
```